README.md

```diff
@@ -73,4 +75,39 @@ func main(){
     // output: [info] ID. name: my name; age: 45; address: address object
 
 }
+```
+
+
+#### Using otel
+
+Once you add this code you can start adding spans and use the zap logger to send events attached to spans.
+* spans can be created as [manual instrumentation](https://opentelemetry.io/docs/instrumentation/go/manual/)
+* or with [instrumentation plugins](https://uptrace.dev/opentelemetry/instrumentations/?lang=go)
+* logs should be attached to a context which contains a span using `.Ctx(ctx)`
+* only logs with severity > Warn will send events
+* the variable `OTEL_COLLECTOR_SVC` configures where to send otel data with the gRPC protocol
+* you can specify `ACCOUNT_ID` to enrich data with it
+
+```go
+package main
+
+import (
+	logger "github.com/kubescape/go-logger"
+	"go.opentelemetry.io/otel"
+)
+
+func main() {
+    // configure otel
+    ctx := logger.InitOtel(logger.L(), "<service>", "<version>")
+    defer logger.ShutdownOtel(ctx)
+
+	// create a span
+	ctx, span := otel.Tracer("").Start(ctx, "<name of the span>")
+	defer span.End()
+
+	if err := cmd.Execute(ctx); err != nil {
+		// attach log to the span
+		logger.L().Ctx(ctx).Fatal(err.Error())
+	}
+}
```

Comment:
question: maybe we should include this in the package-level docs?

I know there are a lot of README files here and there, but none of them show up in the Go docs, and especially not when you hover over a package or a function in your IDE. I think it would help users access our documentation faster if we followed traditional GoDoc conventions.

What do you think?

Reply:
good point, I'll add that